index.ts

```diff
@@ -14,56 +19,71 @@ function getSubscriptionId(): string {
   return subscriptionId!;
 }
 
-async function getAzureClients(
-  subscriptionId: string
-): Promise<{
+async function getAzureClients(subscriptionId: string): Promise<{
   computeClient: ComputeManagementClient;
   networkClient: NetworkManagementClient;
 }> {
-  // See https://github.com/Azure/ms-rest-nodeauth#ms-rest-nodeauth- or
-  // https://github.com/Azure/ms-rest-browserauth#ms-rest-browserauth-
-  // to learn about alternative ways of logging into Azure.
-  const credentials = await interactiveLogin();
+  const credentials = new DefaultAzureCredential();
```

Comment:
please use InteractiveBrowserCredential in the case they are using interactiveLogin.